python-frontend/src/main/java/org/sonar/python/types/TypeShed.java

```diff
@@ -73,6 +72,7 @@
   private static final String THIRD_PARTY_3 = "typeshed/third_party/3/";
   private static final String CUSTOM_THIRD_PARTY = "custom/";
   private static final String PROTOBUF = "protobuf/";
+  private static final String PROTOBUF_THIRD_PARTY = "protobuf/stubs/";
```

Comment:
Currently we have stdlib under `protobuf` and third parties under `protobuf/stubs`. All of this is under `types` which currently holds `Typeshed` as well.
What about going for a `protobuf_stdlib`, `protobuf_thid_parties` and `protobuf_custom` all at the same level, for better clarity?

Reply:
works for me! if that's fine I will let you also do that in your PR or I can do it in another separate PR